src/Equinox.Core/Category.fs

```diff
@@ -32,41 +32,39 @@ open System.Threading.Tasks
 /// Store-agnostic baseline functionality for a Category of 'event representations that fold to a given 'state
 [<NoComparison; NoEquality>]
 type Category<'event, 'state, 'context>
-    (   resolveInner: string -> string-> struct (Core.ICategory<'event, 'state, 'context> * string * (CancellationToken -> Task<unit>) voption),
+    (   name,
+        resolveStream: string -> string -> struct (Core.ICategory<'event, 'state, 'context> * string * (CancellationToken -> Task<unit>) voption),
         empty: struct (Core.StreamToken * 'state)) =
+    member val private Name = name
```

Comment:
should this be public or deleted ?

Reply:
what would you use it for?

Reply:
Debugging or logging ? I would personally never use it

Reply:
don't expose it then 😅 one less thing to BC break

Reply:
BC?

Reply:
Backwards Compatibility

Reply:
yes, good point. Just setting people up for disappointment and grumbling!